examples/colab/tf_hub_delf_module.ipynb

```diff
@@ -222,7 +222,8 @@
         "def download_and_resize_image(url, filename, new_width=256, new_height=256):\n",
         "  response = urlopen(url)\n",
         "  image_data = response.read()\n",
-        "  pil_image = Image.open(StringIO(image_data))\n",
+        "  image_data = BytesIO(image_data) if PY3 else StringIO(image_data)\n",
```

Comment:
Is the case distinction really needed? six.BytesIO (just like six.StringIO) maps to StringIO.StringIO in Python 2, says https://pythonhosted.org/six/

Reply:
ah I hadn't seen that I just knew it didn't work on Python 3 and didn't want to break the 2 support